pdns/rfc2136handler.cc

```diff
@@ -826,7 +826,7 @@ int PacketHandler::processUpdate(DNSPacket *p) {
     if (rr->d_place == DNSResourceRecord::ANSWER) {
       int res = checkUpdatePrerequisites(rr, &di);
       if (res>0) {
-        g_log<<Logger::Error<<msgPrefix<<"Failed PreRequisites check, returning "<<RCode::to_s(res)<<endl;
+        g_log<<Logger::Error<<msgPrefix<<"Failed PreRequisites check for "<<rr->d_name.toLogString()<<", returning "<<RCode::to_s(res)<<endl;
```

Comment:
nit, the `operator<<()` for DNSName already calls `toLogString()` for you :)

Reply:
the rest of the file does it in this style too - I'm not going to waste 0.25 kilowatthours of CI and build to fix this one line.